oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/H2StorageConfig.java

```diff
@@ -59,4 +59,16 @@
      * @since 8.2.0
      */
     private int numOfSearchableValuesPerTag = 2;
+    /**
+     * max size per batch execute sql
+     */
+    private int maxSizeOfBatchSql = 2000;
```

Comment:
No configuration requires `@since 8.8.0`

The storage doc and configuration-vocabulary.md should be updated accordingly.

Reply:
ok . i update backend-storage.md and configuration-vocabulary.md